Assets/Src/Scripts/Animator/FaceController.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SeedUnityVRKit {
+  public class FaceController : MonoBehaviour {
+    public GameObject MouthClose;
+    public GameObject MouthSmall;
+    public GameObject MouthMid;
+    public GameObject MouthLarge;
+    public GameObject EyesClose;
+    public GameObject EyesOpen;
+
+    private void SetObjectVisible(GameObject obj, bool flag) {
+      obj.SetActive(flag);
+    }
+
+    public void SetMouth(MouthShape mouthShape) {
+      switch (mouthShape) {
```

Comment:
Looks very redundant. Why not just 

```
SetObjectVisible(MouthClose, mouthShape == MouthShape.Close);
SetObjectVisible(MouthSmall, mouthShape == MouthShape.Small);
SetObjectVisible(MouthMid, mouthShape == MouthShape.Mid);
SetObjectVisible(MouthLarge, mouthShape == MouthShape.Large);
```

Furthermore, you can just define this in an mapping, and just loop through it:

```
foreach (var status in _statusArray) {
  SetObjectVisible(status.gameObject, mouthShape == status.expectedShape);
}